content/applications/websites/livechat.rst

```diff
@@ -246,7 +279,7 @@ Join or leave a channel
 -----------------------
 
 To join a live chat channel, go to the :menuselection:`Live Chat` app and click the :guilabel:`JOIN`
-button on the kanban card for the appropriate channel.
+button on the Kanban card for the appropriate channel.
```

Comment:
totally your preference, but im not sure if it's necessary to mention the Kanban card, as only the kanban cards are displayed on the Livechat main dashboard
```suggestion
button on the desired channel.
```